pkg/cli/cmds/executions.go

```diff
@@ -124,6 +151,36 @@ func executionDelete(c *cli.Context) error {
 	return nil
 }
 
+func delete(c *cli.Context) error {
+	namespace := c.GlobalString("namespace")
+	kubeConfig := c.GlobalString("kubeconfig")
+
+	if len(c.Args()) != 1 {
+		return InvalidArgs{}
+	}
+
+	name := c.Args().First()
+
+	controllers, err := getControllers(kubeConfig, namespace)
+	if err != nil {
+		return err
+	}
+
+	_, err = controllers.executions.Get(namespace, name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
nit: this and the following can be of the form:
`if _, err = controllers.executions.Get(namespace, name, metav1.GetOptions{}); err !=nil `